doc/nrf/ug_thread_ot_memory.rst

```diff
@@ -24,6 +24,14 @@ Moreover, take into account the following considerations:
 * The single protocol sample was configured by setting :kconfig:option:`CONFIG_BT` to ``n``.
 * Values for the :ref:`Thread CLI sample <ot_cli_sample>`, which works with all OpenThread calls, are the highest possible for the OpenThread stack using the master image library configuration.
 
+The tables provide memory requirements for the following device type variants:
+
+* *FTD* - Full Thread Device.
+* *MTD* - Minimal Thread Device.
+
+Some tables also list a *master* variant, which is an FTD with additional features, such as being able to have the *commissioner* or *border router* commissioning roles.
+See :ref:`thread_ot_commissioning` for more information on device types and commissioning roles.
```

Comment:
I'd rather point the user to the `OPENTHREAD_NORDIC_LIBRARY` configuration option https://github.com/nrfconnect/sdk-nrfxlib/blob/main/openthread/Kconfig#L40 for details about the variants.

Reply:
That's a good point. Do you we should point to the Kconfig in addition to the commissioning documentation? Or not refer to the commissioning documentation at all?

Reply:
I would skip the commissioning documentation here.

Reply:
Changed.